### DISCUSSION TITLE
Netflow Processing Killed

### DISCUSSION THREAD

**Discussion by LaysonRC** [phaag/nfsen#26] [Q&A]
Hi, I'm setting "1 day time window" in my graph. 
With "Stat TopN", 
Top 100,
Stat Flow Records, order by flows
bi-directional

It takes around 1 minute and return "Killed"

Statistics timeslot Oct 26 2023 - 08:55 -03 - Oct 27 2023 - 08:40 -03
Flows: 444.4 /s
Packets: 1.1 k/s
Traffic: 6.5 Mb/s

** nfdump -M /opt/nfsen/profiles-data/live/SOURCE  -T  -R 2023/10/26/nfcapd.202310260840:2023/10/27/nfcapd.202310270840 -n 100 -s record/flows -B
nfdump filter:
any
Command line switch -s overwrites -a

Killed
**

Nfdump-1.7.3
NfSen-1.3.11
Ubuntu-22.04.3lts 

Do you have any tip?

**Comment by phaag** [comment]
Killed means, that the kernel kicked you out due to a too high memory consumption. In order to do evaluate your query, you would need to pump all flows in that time window into memory in order to aggregate and sort. It is obviously too much for your system.
If you run this command, you may observer memory consumption by running top.
You may try to increase the kernel memory sickout limit and/or install more memory.

**Reply by LaysonRC** [reply]
I noticed that nfdump is using all memory (8Gb). There is a way to limit it?
